src/Components/TableComponent/DailyVolumnHistoryTable/DailyVolumnHistoryTable.tsx

```diff
@@ -10,30 +10,30 @@ import { MdTrendingFlat } from 'react-icons/md';
 import { DateObject } from 'react-multi-date-picker';
 import { useLocation } from 'react-router-dom';
 
-import { format } from 'date-fns';
+import { format, subDays } from 'date-fns';
 
 
 
 
 const DailyVolumnHistoryTable = () => {
 
-
-    const [dateRange, setDateRange] = useState<DateObject[] | null>(null);
+    const defaultDate = subDays(new Date(), 1);
+    const [date, setDate] = useState<DateObject | null>(new DateObject(defaultDate));
     const [rows, setRows] = useState([]);
 
     const [filters, setFilters] = useState<Filters>({
         page: '1',
         per_page: '20',
```

Comment:
by default it should filter by previous day's date, see example below{
    "page": "1",
    "per_page": "20",
    "created_at_from": "2025-01-19"
}